pkg/version.go

```diff
@@ -1,4 +1,4 @@
-package common
+package pkg
```

Comment:
Seems like `CommitHash` and `BuildTime` are not used and can be removed.

I would suggest for simmplicity also moving
```
	Name 
	Version  
```
In the single place these are used in api.go

Reply:
Ok we should keep those then.
I think it would be better to define a `constant` package to put this file and `constant.go` content instead of using pkg as a package name